packages/extension/test/integration/v2/background_spec.js

```diff
@@ -857,12 +860,36 @@ describe('app/background', () => {
 
           expect(browser.windows.getCurrent).to.be.called
           expect(browser.tabs.remove).to.be.called
+          expect(browser.tabs.create).not.to.be.called
 
           done()
         })
 
         this.server.emit('automation:request', 123, 'reset:browser:tabs:for:next:test')
       })
+
+      // @see https://github.com/cypress-io/cypress/issues/29172
+      describe('firefox 124 and up', () => {
+        beforeEach(() => {
+          global.window.navigator = {
```

Comment:
Can this be reset in an afterEach?

Reply:
it's reset in the `beforeEach` at the start of the testing block https://github.com/cypress-io/cypress/blob/fix_firefox_124/packages/extension/test/integration/v2/background_spec.js#L121